packages/extension-ui/src/Popup/Derive/Derive.spec.tsx

```diff
@@ -125,6 +125,14 @@ describe('Derive', () => {
       return { address: derivedAddress, suri: defaultDerivation } as ResponseDeriveValidate;
     });
 
+    it('has form with submit button', () => {
```

Comment:
Before, we where testing if clicking on a button creates derived account. We can't do this now, since enzyme `simulate('click')` doesn't trigger `submit` on form. I wanted to make sure, that there is a button that's going to serve this purpose. It's the same test path as before, but because of lack of proper tools split in two... kind of :sweat_smile: 

I think snapshot is an overkill, since all I want is `a` button :grin:

Reply:
I can add separate tests for SelectParent and AccountNamePasswordCreation, but I'm not sure if it's worth an effort

Reply:
If you're testing for _submitting experience_, wouldn't it make sense to have the check for the presence of the button in the `correctly creates the derived account` test, right next to the _submit_ action?

Reply:
Or maybe event [this](https://github.com/enzymejs/enzyme/issues/308#issuecomment-255630011) solution which makes `click` actually submit the form. Maybe looks a bit like a hack, but imho enzyme not submitting a form on a button click is _weird_ or maybe even incorrect, so a hack is not unwelcome.

Reply:
`.click` is not a function :cry:

Reply:
Well, we tried.. :P